components/support/rc_crypto/src/hkdf.rs

```diff
@@ -44,6 +44,28 @@ pub fn expand(prk: &hmac::SigningKey, info: &[u8], out: &mut [u8]) -> Result<()>
     Ok(())
 }
 
+pub fn expand_label(
+    secret: &hmac::SigningKey,
+    label: &str,
+    transcript: &[u8],
+    length: usize,
+) -> Result<Vec<u8>> {
+    let mut info: Vec<u8> = Vec::new();
+    let length_u16: u16 = length as u16;
+    info.extend_from_slice(&length_u16.to_be_bytes());
+    let st = format!("tls13 {}", label);
```

Comment:
Given that this hard-codes some details that are particular to TLS, I think it should live inside the pairing-channel crate rather than here in the shared helper.